core/commands/pin/pin.go

```diff
@@ -44,8 +44,8 @@ type PinOutput struct {
 }
 
 type AddPinOutput struct {
-	Pins     []string
-	Progress int `json:",omitempty"`
+	Pins     []string `json:",omitempty"`
```

Comment:
Is the idea here that while the progress is being reported there's no reason to report the empty pins?

Reply:
Yes. Otherwise, it could look like nothing was pinned.